ethpm_types/utils.py

```diff
@@ -103,7 +103,9 @@ def parse_signature(sig: str) -> tuple[str, list[tuple[str, str, str]], list[str
     outputs_maybe = ""
     if len(outsplit) > 1:
         outputs_maybe = outsplit[1]
-    name, remainder = std_sig.split("(")
```

Comment:
I tested this out, and the input is kinda funny, I am getting:

```
(uint256',
```

when I would expected a

```
'(uint256,address)'
```